README.md

```diff
@@ -24,6 +24,10 @@ higher gas price and track its progress.
 
 This package requires for [IWalletTransactionStorage](src/@types/wallet.ts) and [IWalletSourceStorage](src/@types/wallets.ts) interface to be implemented to store sent transactions and available wallets.
 
+### Gas Price
+This package uses [eth gas station api](https://www.ethgasstation.info/) by default and GAS_STATION_API_KEY should be in process.env for it to work.
+It will fallback to provider.estimateGasPrice if gas station estimation does not work.
```

Comment:
gas station api is free but rate limited. It's too expensive for this to buy key. But if you hit rate limit you should fallback to 
provider.estimateGasPrice

Reply:
It is not free anymore from July 1st. Key is free up to some rate llimit same as before, but you need it.

Reply:
https://ethgasstation.info/blog/eth-gas-station-api-will-require-an-api-key-starting-july-1st-2020/

Reply:
:slightly_frowning_face: